README.md

```diff
@@ -12,26 +12,34 @@ for other blockchain protocols implementations.
 
 ## Getting started
 
-### Prerequisites
-
 If it's the first time you boot a `nodeos` node, please review
 https://developers.eos.io/welcome/latest/tutorials/bios-boot-sequence
 and make sure you get a grasp of what this blockchain node is capable.
 
-You might want to have the [eosio.cdt tools](https://github.com/EOSIO/eosio.cdt)
-installed, as well as `cleos` (from [EOSIO/eos](https://github.com/EOSIO/eos)) or
-[eosc](https://github.com/eoscanada/eosc/releases).
+### Requirements
+
+#### Operating System
+* This software runs on Linux or Mac OS X
 
-### Install
+#### dfuse Instrumented nodeos (deep-mind)
+* See [DEPENDENCIES.md](DEPENDENCIES.md) for instructions on how to get an instrumented nodeos binary (_nodeos-dm_)
```

Comment:
Not sure of the point of having `(_nodeos-dm_)` at the end of the sentence. I would also end all bullet line with a dot, just to be consistent with the rest, finally, I think `nodeos` should always be in code-block (backticks).

```suggestion
* See [DEPENDENCIES.md](DEPENDENCIES.md) for instructions on how to get an instrumented `nodeos` binary.
```